Puppetfile

```diff
@@ -28,7 +28,7 @@ mod 'puppetlabs/catalog_preview', :latest
 mod 'puppet/archive', :latest
 mod 'puppetlabs/inifile', :latest
 mod 'graduway/ec2tagfacts',
-  :git => 'https://github.com/graduway/ec2tagfacts'
+  :git => 'https://github.com/graduway/ec2tagfacts',
```

Comment:
We should start to specify versions for each module in Puppetfile, the default latest version is ok for demo purposes, but on real environments it's recommended to specify version names, in order not to have bad surprises when deploying modules in different times / places.
This comment applies not only to these new entries but to all the modules listed in the Puppetfile